docs/docs/tutorial-eval.md

```diff
@@ -56,7 +56,8 @@ model = ExtractFruitsModel(model_name='gpt-3.5-turbo-1106',
                           prompt_template='Extract fields ("fruit": <str>, "color": <str>, "flavor": <str>) from the following text, as json: {sentence}')
 sentence = "There are many fruits that were found on the recently discovered planet Goocrux. There are neoskizzles that grow there, which are purple and taste like candy."
 print(asyncio.run(model.predict(sentence))) 
-# note: you can also call `await model.predict(sentence)` within async functions
+# if you're in a Jupyter Notebook, run:
```

Comment:
we have this easy to comment in substitute for the last example but not for the previous 2